chapters/data-analysis.tex

```diff
@@ -1,292 +1,488 @@
+Looking for duplicated entries is usually part of data quality monitoring,
+as is the only other reason to change the raw data during cleaning:
+correcting mistakes in data entry.
+During data quality monitoring, you will inevitably encounter data entry mistakes,
+such as typos and inconsistent values.
+If you don't, you are probably not doing a very good job at looking for them.
+These mistakes should be fixed in the cleaned data set,
+and you should keep a careful record of how they were identified,
+and how the correct value was obtained.
 
-There are a number of tasks involved in cleaning data.
+% Data description ------------------------------------------------------------------
+Note that if you are using secondary data, 
+the tasks described above can likely be skipped.
+However, the last step of data cleaning, describing the data,
+will probably still be necessary.
+This is a key step to making the data easy to use, but it can be quite repetitive.
 The \texttt{iecodebook} command suite, part of \texttt{iefieldkit},
 is designed to make some of the most tedious components of this process,
 such as renaming, relabeling, and value labeling,
 much easier (including in data appending).\sidenote{\url{https://dimewiki.worldbank.org/wiki/iecodebook}}
 \index{iecodebook}
-A data cleaning do-file will load the de-identified data,
-make sure all the fields are named and labelled concisely and descriptively,
-apply corrections to all values in the dataset,
-make sure value labels on responses are accurate and concise,
-and attach any experimental-design data (sampling and randomization)
-back to the clean dataset before saving.
-It will ensure that ID variables\sidenote{\url{https://dimewiki.worldbank.org/wiki/ID_Variable_Properties}} are correctly structured and matching,
-ensure that data storage types are correctly set up
-(including tasks like dropping open-ended responses and encoding strings),
-and make sure that data missingness is appropriately documented
-using other primary inputs from the field.\sidenote{\url{https://dimewiki.worldbank.org/wiki/Data_Documentation}}
-
-Clean data then becomes the basis for constructed (final) datasets.
-While data cleaning typically takes one survey dataset
-and mixes it with other data inputs to create a corresponding
-cleaned version of that survey data (a one-to-one process),
-\textbf{data construction} is a much more open-ended process.
-\index{data construction}
-Data construction files mix-and-match clean datasets
-to create a large number of potential analysis datasets.
-Data construction is the time to create derived variables
-(binaries, indices, and interactions, to name a few),
-to reshape data as necessary;
-to create functionally-named variables,
-and to sensibly subset data for analysis.
+We have a few recommendations on how to use this command for data cleaning.
+First, we suggest keeping the same variable names as in the survey instrument,
+so it's easy to connect the two files.
```

Comment:
Is _connect_ a word we use for this? What about:
* _so it's easy to cross-reference the two files_
* _so it's easy to link the same variable across the two files_
* _so it's easy to compare the same variable across the two files_

Reply:
In c80a095:

>First, we suggest keeping the same variable names as in the survey instrument, so it's straightforward to link data points for a variable to the question that originated them.

Reply:
There is a typo or some missing words:

_First, we suggest keeping the same variable names in **the cleaned data set** as in the survey instrument, so it's straightforward to link data points for a variable to the question that originated them._

Reply:
Added "in the cleaned data set" in 97b2ccd.